Tasks/ServiceFabricUpdateManifests/task.json

```diff
@@ -16,14 +16,26 @@
         "Cmd"
     ],
     "version": {
-        "Major": 1,
-        "Minor": 1,
-        "Patch": 12
+        "Major": 2,
```

Comment:
The SF build definition template needs to be updated as well

Reply:
Yep, already being planned.

---
In reply to: [137859543](https://github.com/Microsoft/vsts-tasks/pull/5076#discussion_r137859543) [](ancestors = 137859543)